clouds/aws/roles/roles.py

```diff
@@ -19,6 +19,36 @@ def iam_client(region=DEFAULT_AWS_REGION):
     return boto3.client(service_name="iam", region_name=region)
 
 
+def get_roles(client=None):
+    """
+    Get all roles from IAM.
+    Args:
+        client: A boto3 client for IAM.
+
+    Returns:
```

Comment:
should be:
```
Returns:
    type: description
```

Reply:
@myakove let me know if the updated docstring is what you are looking for.